processors/base/helpers/__init__.py

```diff
@@ -4,20 +4,28 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
 import re
 import json
 import uuid
 import string
 import logging
 import datetime
 import urlparse
+import csv
+import iso3166
+
+from fuzzywuzzy import process, fuzz
```

Comment:
I've tried to do that after reading your first review, but for some reason I can't figure out (visibility?) I'm unable to do such thing as `import fuzzywuzzy` and then use `fuzzywuzzy.fuzz` or `fuzzywuzzy.process` (am I doing any nonsense?). Besides, the [fuzzywuzzy docs](https://github.com/seatgeek/fuzzywuzzy) actually uses this `from fuzzywuzzy import foo` format to import the sub-packages on its examples. That's why I'm keeping this import format so far. Any thoughts on how to keep the format you desire?